control-plane-operator/controllers/hostedcontrolplane/assets/olm/0000_50_olm_00-catalogsources.crd.yaml.guest

```diff
@@ -0,0 +1,169 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
can we stick to the .yaml extension and may create an inner folder for guest manifests?